packages/flutter/lib/src/widgets/navigator.dart

```diff
@@ -2940,7 +2946,8 @@ class _RouteEntry extends RouteTransitionRecord {
   /// remove as a result of a page update.
   static const int kDebugPopAttemptLimit = 100;
 
-  static final Route<dynamic> notAnnounced = _NotAnnounced();
+  static Route<dynamic> get notAnnounced => _notAnnounced ?? _NotAnnounced();
```

Comment:
this isn't right. Needs to be 
`_notAnnounced ??= _NotAnnounced();`
and 
`static Route<dynamic>? _notAnnounced` should be initialized to null instead of `_NotAnnounced()`
Same issue that the notAnnounced getter will start behaving differently after the first update.